apps/charge-accounts-service/src/operators/operators.controller.ts

```diff
@@ -0,0 +1,62 @@
+import { Body, Controller, Get, Post, UseGuards } from '@nestjs/common'
+import { User } from '@app/accounts-service/users/user.decorator'
+import { UsersService } from '@app/accounts-service/users/users.service'
+import { JwtAuthGuard } from '@app/accounts-service/auth/guards/jwt-auth.guard'
+import { ProjectsService } from '@app/accounts-service/projects/projects.service'
+import { CreateOperatorDto } from '@app/accounts-service/operators/dto/create-operator.dto'
+
+@Controller({ path: 'operators', version: '1' })
+export class OperatorsController {
+  constructor (
+    private readonly usersService: UsersService,
+    private readonly projectsService: ProjectsService
+  ) {}
+
+  /**
+   * Get current operator
+   * @param authOperatorDto
+   * @returns the user and project with public key
+   */
+  @UseGuards(JwtAuthGuard)
```

Comment:
Here you should use separated guard for operators reqs

Reply:
@NikolayRn [adding the simple JWT guard individually into each endpoint where JwtAuthGuard is already in place, it is more efficient to extend it within AuthGuard.](https://github.com/fuseio/fusebox-backend/pull/214/files#r1431038648)